README.md

```diff
@@ -31,6 +31,20 @@ Any executable scripts added to the `/startup/startup.d/` directory will be auto
 #### Certificate Installation
 The image comes pre-installed with a startup script which provides a mechanism to extend the CA certificates which should be trusted.
 
+#### Export File-Based Secrets Script
```

Comment:
@michael-bryson / @dermot-hardy Should I update the readme + release notes of the other base images (https://github.com/CAFapi/opensuse-python3-image etc) with this info as well?

Reply:
Yes that's what we have done before as users will consume the base image they want so its better to have details of all the scripts in each base image readme.

Reply:
If they have an inherited section on `Certificate Installation` then I guess so.